tests/rgbpp/spore/launch/3-create-spores.ts

```diff
@@ -0,0 +1,156 @@
+import { BtcAssetsApiError, genCreateSporeCkbVirtualTx, sendRgbppUtxos } from 'rgbpp';
+import {
+  isMainnet,
+  collector,
+  btcAddress,
+  btcDataSource,
+  btcKeyPair,
+  btcService,
+  CKB_PRIVATE_KEY,
+  ckbAddress,
+} from '../../env';
+import {
+  Hex,
+  appendCkbTxWitnesses,
+  appendIssuerCellToSporesCreate,
+  buildRgbppLockArgs,
+  generateSporeCreateCoBuild,
+  sendCkbTx,
+  updateCkbTxWithRealBtcTxId,
+  RawSporeData,
+} from 'rgbpp/ckb';
+import { transactionToHex, utf8ToBuffer } from 'rgbpp/btc';
+import { getFastestFeeRate, readStepLog, writeStepLog } from '../../shared/utils';
+
+interface SporeCreateParams {
+  clusterRgbppLockArgs: Hex;
+  receivers: {
+    toBtcAddress: string;
+    sporeData: RawSporeData;
+  }[];
+}
+
+// Warning: Before runing this file for the first time, please run 2-prepare-cluster.ts
+const createSpores = async ({ clusterRgbppLockArgs, receivers }: SporeCreateParams) => {
+  const { retry } = await import('zx');
+  await retry(20, '10s', async () => {
+    const feeRate = await getFastestFeeRate();
+    console.log('feeRate = ', feeRate);
+
+    const ckbVirtualTxResult = await genCreateSporeCkbVirtualTx({
+      collector,
+      sporeDataList: receivers.map((receiver) => receiver.sporeData),
+      clusterRgbppLockArgs,
+      isMainnet,
+      ckbFeeRate: BigInt(2000),
+    });
```

Comment:
### suggestion
Also follow the best practice -> `saveCkbVirtualTxResult` in the tests.

- ref: https://github.com/ckb-cell/rgbpp-sdk/issues/192